### PR TITLE
Add missing aarch64 condition codes

### PR DESCRIPTION
**Pull request by borismol**
### Description/Motivation/Screenshots ###
Added `cc` and `cs` condition codes

### How Has This Been Tested? ###

| Architecture |          Yes/No          | Comments                                  |
| ------------ | :----------------------: | ----------------------------------------- |
| x86-32       | :heavy_multiplication_x: |  |
| x86-64       | :heavy_multiplication_x: |                                           |
| ARM          | :heavy_multiplication_x: |                                           |
| AARCH64      | :heavy_check_mark: |                                           |
| MIPS         | :heavy_multiplication_x: |                                           |
| POWERPC      | :heavy_multiplication_x: |                                           |
| SPARC        | :heavy_multiplication_x: |                                           |
| RISC-V       | :heavy_multiplication_x: |                                           |
| `make tests` | :heavy_check_mark: |                                           |

### Checklist ###

- [x] My PR was done against the `dev` branch, not `master`.
- [x] My code follows the code style of this project.
- [] My change includes a change to the documentation, if required.
- [] My change adds tests as appropriate.
- [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -1287,7 +1287,10 @@ def gdb_disassemble(start_pc, **kwargs):
         asm = insn["asm"].rstrip().split(None, 1)
         if len(asm) > 1:
             mnemo, operands = asm
-            operands = operands.split(",")
+            operands_and_comment = operands.split("//")
```
Comment (Grazfather) [inline gef.py:1290]: Mind splitting this into another PR? I think this part can be improved.

File: gef.py [hunk 2]
```diff
@@ -1821,6 +1824,8 @@ def is_branch_taken(self, insn):
             taken, reason = val&(1<<flags["carry"]) and not val&(1<<flags["zero"]), "C && !Z"
         elif mnemo.endswith("ls"):
             taken, reason = not val&(1<<flags["carry"]) or val&(1<<flags["zero"]), "!C || Z"
+        elif mnemo.endswith("cs"): taken, reason = bool(val&(1<<flags["carry"])), "C"
+        elif mnemo.endswith("cc"): taken, reason = not val&(1<<flags["carry"]), "!C"
```
Comment (Grazfather) [inline gef.py:1828]: Looks good.

File: gef.py (hunk 1 above)
Comment (borismol) [reply gef.py]: Thank you for review. Ok. I reverted that.

**Comment by hugsy** [comment]
It looks good to me too, but I would like to test this PR on a proper AARCH64 environment to be 100% sure. I'll try to do that asap.

File: gef.py [hunk 3]
```diff
@@ -1789,7 +1789,7 @@ def flag_register_to_human(self, val=None):
         return flags_to_human(val, self.flags_table)
 
     def is_conditional_branch(self, insn):
-        conditions = {"eq", "ne", "lt", "le", "gt", "ge", "vs", "vc", "mi", "pl", "hi", "ls"}
+        conditions = {"eq", "ne", "lt", "le", "gt", "ge", "vs", "vc", "mi", "pl", "hi", "ls", "cs", "cs"}
```
Comment (daniellimws) [inline gef.py:1792]: `cs` and `cc` right?

**Review by daniellimws — CHANGES_REQUESTED** [review]
Typo in the `conditions` set?

File: gef.py (hunk 3 above)
Comment (Grazfather) [reply gef.py:1792]: 🦅 👁️

Comment (borismol) [reply gef.py]: Thank you! Fixed.

**Review by daniellimws — APPROVED** [review]
👍

**Review by hugsy — APPROVED** [review]
Tested and all works out

**Pull request merged**